docker-compose.yml

```diff
@@ -40,14 +41,33 @@ services:
       OAUTH__CLIENTID: ${OAUTH__CLIENTID}
```

Comment:
I dont think any of the these environment variables are required on build are they?  Does this set the base environment in the container?  If so, we shouldn't be building images using fake service.

Reply:
no, they are not needed for build.